clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -491,8 +611,10 @@ message SearchRequest {
     bool explain = 25;
     // Search nested object fields for each hit
     map<string, InnerHit> inner_hits = 26;
+    // Defines runtime fields for this query
+    repeated RuntimeField runtimeFields = 27;
     // Request kNN vector search queries, results will be combined with the standard query (if provided) using the boolean query SHOULD logic
-    repeated KnnQuery knn = 27;
+    repeated KnnQuery knn = 28;
```

Comment:
The field number for knn had to be changed to 28 since 27 is used by runtimeFields.